content/operate/rc/supported-regions.md

```diff
@@ -0,0 +1,140 @@
+---
+Title: Supported Cloud providers and regions
+alwaysopen: false
+categories:
+- docs
+- operate
+- rc
+description: null
+linktitle: Supported regions
+weight: 90
+---
+
+Choosing your cloud provider and region may affect latency between your application and your database, and may affect what connectivity options are available for your database.
+
+Redis Cloud supports databases on the following cloud providers:
+- [Amazon Web Services (AWS)](#amazon-web-services)
+- [Google Cloud](#google-cloud)
+- [Microsoft Azure](#microsoft-azure)
+
+## Amazon Web Services
+
+Redis Cloud supports databases on all of the following Amazon Web Services (AWS) regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. 
```

Comment:
This suggestion might not be correct if a "region" is understood to mean a server more than a geographical region.
```suggestion
Redis Cloud supports databases in all of the following Amazon Web Services (AWS) regions. All regions support Redis Cloud Pro databases, but only certain regions support Redis Cloud Essentials databases. 
```

Reply:
```suggestion
Redis Cloud supports databases in the following Amazon Web Services (AWS) regions. 
```